torch/functional.py

```diff
@@ -1291,5 +1293,112 @@ def _lu_no_infos(A, pivot=True, get_infos=False, out=None):
     func_name='lu')
 lu.__doc__ = _lu_impl.__doc__
 
+def clamp(input, min=None, max=None, out=None):
+    r"""
+    Clamp all elements in :attr:`input` into the range `[` :attr:`min`, :attr:`max` `]` and return
+    a resulting tensor:
+
+    .. math::
+        y_i = \min(\text{max}_i, \min(\text{min}_i, x_i))
+
+    The :attr:`input` and arguments :attr:`min` and attr:`max` should have matching types.
+    I.e. if :attr:`input` is of type `FloatTensor` or `DoubleTensor`, then args :attr:`min`
```

Comment:
The terms "FloatTensor" and DoubleTensor" are deprecated. I think this sentence can just be cut, anyway.